_Group Projects/Group_5/Week8_Assignment/Group5_Week8_Simulation.do

```diff
@@ -0,0 +1,115 @@
+///////////////////////////////
+///*** WEEK 8 SIMULATION ***///
+///////////////////////////////
+
+*Group 5: Noah Blake Smith, Miglé Petrauskaité, and Benjamin Tu
+
+*Date: October 30, 2022
+
+/*
+Description:
+
+Each row is a child who applied to the DC charter lottery.
+*/
+
+clear all
+
+cap program drop charter_simulation
+
+program define charter_simulation, rclass // Define r-class program
+
+set obs 5000 // Approximate number of children who apply to PK3 lottery
```

Comment:
Perhaps you could make this an input so you can see how changing the sample size affects your results?